docs/source/pypureclient.rst

```diff
@@ -22,6 +22,14 @@ pypureclient.api\_token\_manager module
    :undoc-members:
    :show-inheritance:
 
+pypureclient.client\_settings module
```

Comment:
I don't think this change is needed

Reply:
It is autogenerated, I think it would be great to have more customisation on docs generating.